CONTRIBUTING.rst

```diff
@@ -149,22 +148,21 @@ You may need to run ``conda install cmake`` for the test requirements to properl
 
 Ensure `Docker <https://www.docker.com/>`_ is installed.
 
-``npm`` is required to run the Javascript dev server and the tracking UI.
-You can verify that ``npm`` is on the PATH by running ``npm -v``, and
-`install npm <https://www.npmjs.com/get-npm>`_ if needed.
+Finally, we use ``pytest`` to run all Python contributed code. Install ``pytest``:
 
-If contributing to MLflow's R APIs, install `R <https://cloud.r-project.org/>`_. For changes to R
-documentation, also install `pandoc <https://pandoc.org/installing.html>`_ 2.2.1 or above,
-verifying the version of your installation via ``pandoc --version``. If using Mac OSX, note that
-the homebrew installation of pandoc may be out of date - you can find newer pandoc versions at
-https://github.com/jgm/pandoc/releases.
+.. code-block:: bash
 
-If contributing to MLflow's Java APIs or modifying Java documentation,
-install `Java <https://www.java.com/>`_ and `Apache Maven <https://maven.apache.org/download.cgi>`_.
+    pip install pytest
 
+JavaScript and UI
+~~~~~~~~~~~~~~~~~
+
+``npm`` is required to run the Javascript dev server and the tracking UI.
```

Comment:
For completeness, can we add an introduction like : 
The MLflow UI is written in JavaScript. `npm` ...

Reply:
That sounds reasonable.